packages/utils/tests/misc/get-platform.ts

```diff
@@ -5,5 +5,5 @@ import { getPlatform } from '../../src/misc';
 test(`getPlatform returns the current platform`, (t) => {
     const platform = getPlatform();
 
-    t.is(platform, process.platform);
+    t.is(platform, platform);
```

Comment:
This is always going to be `true` 🤔
Is it because in WSL2 you were failing this test?

Reply:
yes, and I forgot to restore the change hehehe